src/sentry/models/relay.py

```diff
@@ -10,13 +10,25 @@
 from sentry_relay import PublicKey
 
 
+class RelayUsage(Model):
```

Comment:
Should this have a relation to `Relay` instead of duplicating the `relay_id`?

Reply:
I don't think a foreign key would be the best choice here. `RelayUsage` is essentially used as a historical record of what has run. The `Relay` table is more a record of which `Relay` has registered and can now send queries. While in theory having a foreign key from RelayUsage to Relay should work I think it would be limiting. We could remove all records from the `Relay` table (and force all managed Relay to re-register) and that should not affect our historical record of what Relay were run and when.